### ISSUE TITLE
[Question] About the `next step`

### ISSUE THREAD

**Issue by QIN2DIM** [QIN2DIM/hcaptcha-challenger#31]
Recently I have received many letters from developers. I apologize in advance for not responding in time.

## 💖 So here is a unified reply

I'd like to talk about my initial reason for doing this project. As you have seen in the [wiki documentation](https://github.com/QIN2DIM/hcaptcha-challenger/wiki/Advanced), the project's scaffold interface is named `demo`. Yes, I started out to build a shelf to demonstrate this solution, with the fundamental goal of providing a more visual way to demonstrate its speed and pass rate. **However**, it is clear that people are more concerned about whether they can obtain effective `TOKEN` after solving the hCaptcha challenge. **Or** how to embed this solution in 「my」project.

I honestly didn't expect so many people to follow this project, so obviously the portability of this project is very poor at the moment, especially across development languages.

If you've been following my recent work, you'll see that I've been using `hcapotcha-challegner` in the [epic-awesome-gamer](https://github.com/QIN2DIM/epic-awesome-gamer) to handle hCaptcha, and it works perfectly on all major operating systems. So I can understand how developers in various fields need a solution to handle the human-machine challenge.Especially the open source free:)

## TL;DR 

so, about the `next step`. I will continue to refactor this project until it meets the needs of some Python developers. Then, I hope it can be ported to various technology stacks.

As of now, the barrier to entry is extremely high for developers not familiar with python or automation testing (selenium). So I also intend to be requirements-oriented and adapt the technical model of this project so that it can be called in a simpler way.

Finally. My ability is limited, so please correct me if there are any errors or omissions in communication or development.

Have fun~🕶

**Comment by ChronoBrake** [comment]
Nice to hear that,
Waiting for future updates
Cheers

**Comment by ChronoBrake** [comment]
Any ETa next update or updates plans? @QIN2DIM 
Regards
